<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="input_bt">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>输入路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="output_bt">
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>输出路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="state_bt">
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>消除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="export_bt">
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>导出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>画笔大小</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="pen_size_sd">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pen_size_lb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>20</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="file_pb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_bx">
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>AUTO</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="next_bt">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>上一张</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filename_lb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="past_bt">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>下一张</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
        <item>
         <widget class="ScollWidget" name="draw_lb" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>320</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ScollWidget" name="resut_lb" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>320</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="input_lb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>请选择输入文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="output_lb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>请选择输出文件夹</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScollWidget</class>
   <extends>QWidget</extends>
   <header>scollwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pen_size_sd</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pen_size_lb</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
